addons/l10n_ee/migrations/1.2/pre-migrate.py

```diff
@@ -0,0 +1,12 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+
+    if not env.ref('l10n_ee.tax_report', raise_if_not_found=False):
+        cr.execute("""
+            UPDATE ir_model_data
+               SET name = 'tax_report'
+             WHERE module='l10n_ee' AND name='tax_report_vat'
+        """)
```

Comment:
There is no need to pre-check, just run the update query.
```suggestion
    cr.execute("""
        UPDATE ir_model_data
           SET name = 'tax_report'
         WHERE module='l10n_ee' AND name='tax_report_vat'
    """)
```